Source/AwsCommonRuntimeKit/crt/Utilities.swift

```diff
@@ -272,3 +272,18 @@ func withByteCursorFromStrings<Result>(
         }
     }
 }
+
+func withByteCursorArrayFromStringArray<R>(
+    _ arg: [String], _ body: (UnsafePointer<aws_byte_cursor>, Int) -> R) -> R {
```

Comment:
Trivial: Would be cleaner to do 
```suggestion
extension Array where Element == String {
  func withByteCursorArray<R>(
      _ arg: [String], _ body: (UnsafePointer<aws_byte_cursor>, Int) -> R) -> R {
``` 
and then you can do string.withByteCursorArray ...